pyfranca/ast.py

```diff
@@ -339,6 +345,15 @@ def __init__(self, name, key_type, value_type):
         self.value_type = value_type
 
 
+class Constant(ComplexType):
```

Comment:
Also it seems to me that we need a family of AST representations for constants, inheriting from `Constant`. E.g. `StringConstant`, `IntegerCosntante`, etc. These would eventually help with "casting" or implementing basic "type system rules" as explained in Franca's user guide 0.9.2, section 5.3.1 .